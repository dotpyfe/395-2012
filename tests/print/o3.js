var ; print({"x":"y", "z":"w", "abcdefghijklmnopqrtsuvwxyz":"abcdefghijklmnopqrtsuvwxyz"})
