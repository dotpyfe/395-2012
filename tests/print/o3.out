{"x":"y", "z":"w", "abcdefghijklmnopqrtsuvwxyz":"abcdefghijklmnopqrtsuvwxyz"}
